app/components/home-header.tsx

```diff
@@ -26,11 +26,11 @@ const HomeHeader = ({
       paddingTop={{ base: "57px", md: "76px", xl: "78px" }}
     >
       <Box
-        w={{ base: "base", xl: "xl" }}
+        w={{ base: "full", xl: "7xl" }}
```

Comment:
use Chakra's existing "full" space token for 100% and "7xl" token for 1280px (but in ems)